Dockerfile.test.php7

```diff
@@ -1,4 +1,4 @@
-FROM php:7.3
+FROM php:7.4
```

Comment:
This shouldn't be required since the CI replaces it.

Reply:
It is required to run the test on a local VM (outside Github) for debugging purpose.

Reply:
Isn't that what Dockerfile.test.php7.debug is for?

Reply:
From the [documentation](https://github.com/aces/Loris/tree/main/test) I understand that Dockerfile.test.php7.debug is used when XDebug is required (npm run tests:integration:debug) and Dockerfile.test.php7 is used otherwise (npm run tests:integration). Let me know if my assumption is wrong.